guides/assets/esa-initial-flow.txt

```diff
@@ -0,0 +1,40 @@
+sequenceDiagram
+    participant User
+    participant App
+    participant Application Route
+    participant Authenticated Route
+    participant Unauthenticated Route
+    participant Session Service
+    participant Torii Provider
+    participant GitHub
+    participant Token Exchanger
+    User->>App: Use https://git-tool.com
+    App->>Application Route: transitionTo(application)
```

Comment:
This `transitionTo` doesn't actually happen. Instead this is the application route's `beforeModel`.